### PR TITLE
Admin audio cue toggle.

### PR DESCRIPTION
**Pull request by realforest2001**
# About the pull request
Changes the toggle for ARES interface pings to also include prayers (and renames it due to this)
Also moved it from logs category to sound, because it's for sounds...
Added SPAN_BIGNOTICE which combines bold & big classes to get rid of "<b><big></big></b>"

# Explain why it's good for the game

Admin QOL, the sound can be overly intrusive and larger text gets the point across.

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Test to confirm it works as intended.

</details>

# Changelog
:cl:
code: Added a new span class that combines bold and big.
code: Tweaked the way prayer sends notifications to be more efficient.
admin: Moved the prayer notification sound to a toggle preference, combined with ARES Interface notifications.
/:cl:

### CODE REVIEW

File: code/modules/admin/admin_verbs.dm [hunk 1]
```diff
@@ -585,15 +585,15 @@ GLOBAL_LIST_INIT(roundstart_mod_verbs, list(
 	message_admins("[key_name(usr)] announced a random fact.")
 	SSticker.mode?.declare_fun_facts()
 
-/client/proc/toggle_ares_ping()
-	set name = "Toggle ARES notification sound"
-	set category = "Preferences.Logs"
+/client/proc/toggle_admin_pings()
+	set name = "Toggle StaffIC log sounds"
```
Comment (fira) [inline code/modules/admin/admin_verbs.dm:589]: aaa

Comment (realforest2001) [reply code/modules/admin/admin_verbs.dm:589]: wut

Comment (realforest2001) [reply code/modules/admin/admin_verbs.dm:589]: I don't understand :sob:

Comment (fira) [reply code/modules/admin/admin_verbs.dm:589]: Sorry i just meant it's lacking a space, whatever

**Review by fira — APPROVED** [review]

**Pull request merged**